Misc/NEWS.d/next/Library/2021-11-07-15-31-25.bpo-23041.564i32.rst

```diff
@@ -0,0 +1,2 @@
+Add QUOTE_STRINGS and QUOTE_NOTNULL to the suite of csv module quoting
+styles.
```

Comment:
```suggestion
Add ``QUOTE_STRINGS`` and ``QUOTE_NOTNULL`` to the suite of
:mod:`csv` module quoting styles.
```

Reply:
Or these could use this markup to create links:
```
:data:`~csv.QUOTE_STRINGS`
```
The tilde is to hide `csv` in the output, but it’s still there to create the right link.